app/localization/translated/zh.json

```diff
@@ -2125,7 +2125,7 @@
   "Widgets.Description.componentHealthCheck": "显示由指定属性所指示的应用程序组件的通过率。",
   "Widgets.Description.componentHealthCheckTable": "显示由指定属性所指示的应用程序组件的详细统计信息。由于小部件包含复杂的查询，小部件可能需要渲染一段时间（大约10分钟）",
   "Widgets.Description.cumulative": "显示具有相同属性关键字的测试任务的汇总统计的增长趋势。",
-  "Widgets.Description.flakyTestCases": "显示在指定的先前测试任务中前50名最不稳定的测试用例。",
+  "Widgets.Description.flakyTestCases": "显示指定测试任务中最不稳定的前50个测试用例。",
```

Comment:
Let's not translate to Chinese via translator. Please in case of changes in such messages leave default message in English.

Reply:
Done.